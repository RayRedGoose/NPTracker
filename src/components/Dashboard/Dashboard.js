import './Dashboard.scss'
import React from 'react'

const Dashboard = () => {
  return (
    <section className="dashboard">
      <h2>Please, click on tab to see information</h2>
    </section>
  )
}

export default Dashboard
