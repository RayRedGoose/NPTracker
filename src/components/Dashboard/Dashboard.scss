@import "../../variables.scss";

.dashboard {
  h2 {
    color: $accent-color-primary-dark;
    font-family: Salsa;
    letter-spacing: 0.03em;
    padding: 20px;
  }
}
