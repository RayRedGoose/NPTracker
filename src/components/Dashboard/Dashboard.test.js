import React from 'react'
import { shallow } from 'enzyme'
import Dashboard from './Dashboard'

describe('Dashboard', () => {
  it('should match the snapshot with all data passed in correctly', () => {
    const board = shallow(<Dashboard />)

    expect(board).toMatchSnapshot()
  })
})
