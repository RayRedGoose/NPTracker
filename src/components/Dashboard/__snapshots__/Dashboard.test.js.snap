// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dashboard should match the snapshot with all data passed in correctly 1`] = `
<section
  className="dashboard"
>
  <h2>
    Please, click on tab to see information
  </h2>
</section>
`;
